imagej/imagej-ops2/src/main/java/net/imagej/ops2/filter/gauss/Gaussians.java

```diff
@@ -71,11 +70,11 @@ public class Gaussians {
 	 *           type='org.scijava.function.Computers$Arity4'
 	 */
 	@SuppressWarnings({ "unchecked", "rawtypes" })
-	public static <T extends NumericType<T> & NativeType<T>> void defaultGaussRAI(
-		final RandomAccessibleInterval<T> input, //
+	public static <S extends NumericType<S>, T extends NumericType<T>> void defaultGaussRAI(
+		final RandomAccessibleInterval<S> input, //
```

Comment:
How about using `EI` for element type of input image, and something else consistent... `EB`?... for element type of `OutOfBoundsFactory`? We use `EI` for element type of input images and `EO` for element type of output images in ImageJ Ops v1 in a few places, and I'd like to see more consistency here if possible, to reduce possible confusion.

Reply:
A quick grep search shows no instances of `EI`/`EB`/`EO` in the incubator, so I'd prefer to not introduce it.

I used `S` and `T` in this class because that's what the underlying `SeparableSymmetricConvolution` uses, but we could change it to `I`/`O`!